### PR TITLE
Add Products to backend and GET using API

### LINKED ISSUE #6: Add Products to backend, call API on frontend
1. Add products for all categories to the mockbee category database
2. Implement GET request to fetch the categories and their respective products

User should be able to see either all the product cards, based on the category they select.

### PR DESCRIPTION
**Pull request by Hamdrive**
## Overview
This PR refers to issue #6 

The product page has been successfully created, and renders as expected

Features/Changes:
1. Products added to backend (5 products for each category)
2.  GET request using products API and rendering data in cards accordingly
3.  Ensure proper page render to fill up space while data is fetched from backend

## Deploy preview
https://deploy-preview-8--fueledup-store.netlify.app/

## Image Preview
image

### CODE REVIEW

File: src/pages/login/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                  type="submit">
+                  Login
+                </button>
+              </form>
+            </div>
+          </section>
+        </main>
+      </div>
+    </div>
+                    </>
```
Comment (Ronak-Mutha) [inline src/pages/login/Login.jsx:77]: Wierd indentation. Please check and do needful.

**Review by Ronak-Mutha — COMMENTED** [review]
Except for the indentation inconsistency change suggested due to extra element probably, good to merge PR

**Review by GuptaEesh — COMMENTED** [review]
>Good use of classes.
Good use of random for rating.
Descriptive class names used

Good to merge.

File: src/pages/login/Login.jsx (hunk 1 above)
Comment (Hamdrive) [reply src/pages/login/Login.jsx:77]: Fixed it in local, will push this change to Login branch once created

**Comment by Hamdrive** [comment]
Successfully reviewed PR by 2 members. No further conflicts have arisen.
This fixes issue #6 and branch will be merged to dev, closing PR

**Pull request merged**